### PR TITLE
Added a byline to each post

### PR DESCRIPTION
**Pull request by blakejkaplan**

### CODE REVIEW

File: src/components/PostDetail.js [hunk 1]
```diff
@@ -24,6 +25,12 @@ var PostDetail = React.createClass({
         <Text style={styles.caption}>{post.image.caption}</Text>
       );
     }
+    var byline;
+    if (post.authors){
+      byline = (
+        <Text style={styles.byline}>By {post.authors.toString(" ")}</Text>
```
Comment (themichaellai) [inline src/components/PostDetail.js]: I think [`join()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) would be more appropriate than `toString`

File: src/components/PostDetail.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ var PostDetail = React.createClass({
   },
 
   render: function() {
+    //console.log(this.props.post);
```
Comment (themichaellai) [inline src/components/PostDetail.js]: Remove this

File: src/components/PostDetail.js (hunk 1 above)
Comment (blakejkaplan) [inline src/components/PostDetail.js]: Yeah oops...I'll fix that

**Comment by blakejkaplan** [comment]
I fixed the delimiter

**Pull request merged**